content/docs/developing-operators.md

```diff
@@ -21,15 +21,20 @@ The `operator.yaml` is the main YAML file defining both operator metadata as the
 First let’s create `operator.yaml` and place it in a `first-operator` folder.
```

Comment:
IHMO would great say create the dir `first-operator` and then. 
Provide the steps for the new users to be able to cut and paste would be nice too.

Reply:
thanks! Tracked here https://github.com/kudobuilder/www/issues/97